navigationevent/navigationevent-compose/build.gradle

```
@@ -22,38 +22,36 @@
  * modifying its settings.
  */
 
-import androidx.build.SoftwareType
-import androidx.build.PlatformIdentifier
+import androidx.build.JetbrainsAndroidXPlugin
+import androidx.build.AndroidXComposePlugin
+import androidx.build.Publish
 import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType
-import org.jetbrains.kotlin.konan.target.Family
 
 plugins {
     id("AndroidXPlugin")
+    id("com.android.library")
     id("AndroidXComposePlugin")
+    id("JetbrainsAndroidXPlugin")
 }
 
-androidXMultiplatform {
-    androidLibrary {
-        compileSdk = 36
-        namespace = "androidx.navigationevent.compose"
-    }
+AndroidXComposePlugin.applyAndConfigureKotlinPlugin(project)
+JetbrainsAndroidXPlugin.applyAndConfigure(project)
+
+androidXComposeMultiplatform {
+    android()
     desktop()
-    mac()
-    linux()
-    ios()
-    watchos()
-    tvos()
-    mingwX64()
+    darwin()
     js()
-    wasmJs()
-
-    defaultPlatform(PlatformIdentifier.ANDROID)
+    wasm()
+}
 
+kotlin {
     sourceSets {
         commonMain {
             dependencies {
-                api(project(":navigationevent:navigationevent"))
-                api("androidx.compose.runtime:runtime:1.9.0")
+                api("androidx.navigationevent:navigationevent:1.0.0-alpha07")
```

```suggestion
                def version = project.findProperty('artifactRedirection.version.androidx.navigationevent')
                api("androidx.navigationevent:navigationevent:$version")
```

(please use the same in other places too)